### PR TITLE
docs: improve getting started example

### LINKED ISSUE #11: `scheduler` is undefined
## 🐛 Bug Report

Can't access `schedule` object

## To Reproduce

See attachments

## Expected behavior

I expect it to work

## Your Environment

- *node version*: 14.16.0
- *fastify version*: >= 3.14.2
- *fastify schedule v*: >= 1.0.0
- *os*: Linux
- *any other relevant information*
-

### PR DESCRIPTION
**Pull request by is2ei**
Closes #11 
I tested the example code in StackBlitz.
https://stackblitz.com/edit/node-str32w?file=index.js

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,25 +20,24 @@ npm i fastify-schedule toad-scheduler
...
-    () => { return db.pollForSomeData().then((result) => { /* continue the promise chain */ }) },
-    (err: Error) => { /* handle errors here */ }
-)
-const job = new SimpleIntervalJob({ seconds: 20, }, task)
 
-fastify.scheduler.addSimpleIntervalJob(job)
+// `fastify.scheduler` becomes available after initialization.
+// Therefore, you need to call `ready` method.
+fastify.ready().then(() => {
+    const task = new AsyncTask(
```
Comment (kibertoad) [inline README.md:32]: I would extract task and job definition outside of `ready` block, as they do not rely on app being up already

Comment (is2ei) [reply README.md:32]: @kibertoad 
Thanks for your comment. I will fix it.

Comment (is2ei) [reply README.md]: @kibertoad 
I fixed it. Could you review it again? 🙇 
https://github.com/fastify/fastify-schedule/pull/79/commits/053691f71998112982ccb37d4d1db6f4b57069d6

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
@Fdawgs seems like we have broken CI somewhere along the way

**Comment by kibertoad** [comment]
thank you!

**Pull request merged**